packages/core/bootstrap/src/lib/provider-limits/limits.json

```
@@ -565,9 +565,12 @@
   },
   "ethgasstation": {
     "http": {
-      "free": {
-        "rateLimit1m": 120,
+      "trial": {
+        "rateLimit1h": 1.38,
         "note": "could be off, no mention on their site but found here: https://github.com/ethgasstation/ethgasstation-api/blob/ec36f7dd78fccbb6a67ddffc0e12ac1150db8e80/settings.docker.conf"
+      },
+      "starter": {
+        "rateLimit1h": 1388.88
```

I would use 31 days for the calculation as worst-case scenario, and link to the API pricing page which says the limits:
```suggestion
      "trial": {
        "rateLimit1h": 1.34,
        "note": "Taken from the API key limit as 1,000 / 31 / 24 (EGS uses Defi Pulse's API keys): https://data.defipulse.com/#pricing"
      },
      "starter": {
        "rateLimit1h": 1344.08,
        "note": "Taken from the API key limit as 1,000,000 / 31 / 24 (EGS uses Defi Pulse's API keys): https://data.defipulse.com/#pricing"
```

---

updated the rate limits in the new config folder `packages/core/bootstrap/src/lib/config/provider-limits/limits.json`